net-libs/libhubbub/libhubbub-0.3.6.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs flag-o-matic
+
+DESCRIPTION="HTML5 compliant parsing library, written in C"
+HOMEPAGE="https://www.netsurf-browser.org/projects/hubbub/"
+SRC_URI="https://download.netsurf-browser.org/libs/releases/${P}-src.tar.gz"
+
+LICENSE="MIT"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~arm ~ppc ~ppc64 ~x86 ~m68k-mint"
+IUSE="doc test"
+
+BDEPEND="virtual/pkgconfig
+	dev-util/netsurf-buildsystem
+	test? ( dev-lang/perl )"
+RDEPEND=">=dev-libs/libparserutils-0.2.1-r1"
```

Comment:
This has a changing soname, if it gets linked during compile, you must add a subslot binder here. So append with `:=` if needed. Also the version requirement can be lifted off, because it will always be satisified in ::gentoo.

Reply:
This really is something pkgcheck should pick up imho.  Well spotted.

Reply:
It's really hard for an automated check like that to confirm. Needs building libraries and binaries, and see how/where they are linked.